styles.css

```diff
@@ -1,3 +1,1317 @@
+    background-repeat: no-repeat;
+    cursor: pointer;
+    background-position: center;
+}
+
+.details-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 4px;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 36px 12px 36px;
+    border: none;
+    cursor: pointer;
+    background-color: transparent;
+    border: 1px solid #21E786;
+    color: #fff;
+}
+
+.whitelist,
+.read-more {
+    color: #fff;
+    background-color: transparent;
+    border: 2px solid #21E786;
+}
+
+.discord {
+    color: #fff;
+    background-color: #141B22;
+}
+
+.connect,
+.connect-wallet,
+.buy,
+.joc-whitelist {
+    color: #141B22;
+    background: #21E786;
+}
+
+.join-discord,
+.join-twitter {
+    color: #fff;
+    background-color: transparent;
+    border: 2px solid #888B8E;
+}
+
+.joc-discord {
+    color: #fff;
+    background-color: #5865F2;
+}
+
+.breadcrumbs-nav a {
+    display: flex;
+    font-size: 14px;
+    color: #888B8E;
+    font-weight: 400;
+    line-height: 20px;
+}
+.breadcrumbs-nav a:first-of-type::after{
+    content:'';
+    position: relative;
+    left: 10px;
+    top: 2px;
+    width: 2px;
+    height: 16px;
+    background-image: url("data:image/svg+xml,%3Csvg width='3' height='16' viewBox='0 0 3 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='0.5' width='2' height='16' fill='%23888B8E'/%3E%3C/svg%3E ");
+    background-repeat: no-repeat;
+}
+.breadcrumbs-nav a[href="blog-details.html"]::before {
+    content:'';
+    position: relative;
+    right: 10px;
+    top: 2px;
+    width: 2px;
+    height: 16px;
+    background-image: url("data:image/svg+xml,%3Csvg width='3' height='16' viewBox='0 0 3 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Crect x='0.5' width='2' height='16' fill='%23888B8E'/%3E%3C/svg%3E ");
+    background-repeat: no-repeat;
+}
+
+.main-heading,
+.heading {
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    color: #fff;
+    margin: 0;
+}
+
+.sub-heading,
+.how-to-text {
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #fff;
+}
+
+.how-to-heading {
+    font-size: 22px;
+    font-weight: 400;
+    line-height: 30px;
+    text-align: center;
+    color: #21E786;
+
+}
+
+.ot-member-pos,
+.faq-description,
+.joc-text,
+.sub-text .sub-info {
+    font-family: Poppins, Arial, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.copyright-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+
+}
+
+.sr-only {
+    border: 0;
+    clip: rect(1px, 1px, 1px, 1px);
+    -webkit-clip-path: inset(50%);
+    clip-path: inset(50%);
+    height: 1px;
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute;
+    width: 1px;
+    white-space: nowrap;
+}
+
+ul,
```

Comment:
Tag styles should be written at the beginning of the file since they are common across the entire site.